blueprints/controllers/ikea_e1744/ikea_e1744.yaml

```
@@ -142,8 +157,6 @@ variables:
   click_short: '{{ actions_mapping[integration_id]["click_short"] }}'
   click_double: '{{ actions_mapping[integration_id]["click_double"] }}'
   click_triple: '{{ actions_mapping[integration_id]["click_triple"] }}'
-  # integrations which need to store the previous long press event to determine which button was released
-  integrations_with_prev_event_storage: [zha, zigbee2mqtt]
   # build data to send within a controller event
```

```suggestion
  # integrations which need to store the previous long press event to determine which button was released
  integrations_with_prev_event_storage: [zha, zigbee2mqtt]
  # build data to send within a controller event
```

We have to keep the `integrations_with_prev_event_storage` to identify whether the left or right rotation has stopped.

---

We do this anyway. But because we now use the helper for everything we don't need to define what integration uses it since they all use the helper and thus zha and zigbee2mqtt have this per definition.

---

The `integrations_with_prev_event_storage` allows to avoid an additional condition in the `rotate_stop` actions for deCONZ users.